services/movie_session.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import Optional
+from django.db.models import QuerySet
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    movie = Movie.objects.get(pk=movie_id)
+    cinema_hall = CinemaHall.objects.get(pk=cinema_hall_id)
+    return MovieSession.objects.create(
+        movie=movie,
+        cinema_hall=cinema_hall,
+        show_time=movie_show_time
+    )
+
+
+def get_movies_sessions(session_date: Optional[str] = None) -> QuerySet:
+    queryset = MovieSession.objects.all()
+    if session_date:
+        queryset = queryset.filter(show_time__date=session_date)
+    return queryset
+
+
+def get_movie_session_by_id(movie_session_id: int) -> Optional[MovieSession]:
+    try:
+        return MovieSession.objects.get(pk=movie_session_id)
```

Comment:
This is not so good pattern - if record doesn't exist, warning should be raised and execution must be stopped, if you return None instead of a MovieSession it may lead to unpredictable behavior in other parts of the program